resolve/resolve_test.go

```diff
@@ -16,14 +16,14 @@ import (
 
 // A test may enable non-standard options by containing (e.g.) "option:recursion".
 func getOptions(src string) *syntax.FileOptions {
-	// TODO(adonovan): use new fine-grained names.
```

Comment:
The intent of this TODO was to get rid of the test option `globalreassign` and express it in terms of the fine-grained flags. When the TODO is done, every field in getOptions' FileOptions will be set by exactly one `option(src, "...")` call. Would you care to make that change?

Reply:
Ok, I think I did that. I removed the new tests from `resolve.star` as that functionality was already tested in others. I updated the flags in the tests according to this suggestion.